.github/workflows/build.yml

```diff
@@ -133,7 +133,7 @@ jobs:
         # Decrypt the certificate
         openssl enc -aes-256-cbc -d -md sha512 \
           -k "${MACOS_CODESIGN_OPENSSL_PASSWORD}" \
-          -in /Users/runner/work/mixxx/mixxx/cmake/macos_developer_id_codesign_certificate.p12.enc \
+          -in /Users/runner/work/mixxx/mixxx/cmake/certificates/macos_developer_id_codesign_certificate.p12.enc \
```

Comment:
This should be moved to packages or better moved to GitHub.
The cmake folder should IMHO be free from mixxx specific stuff.